docs/reference/urllib3.response.rst

```diff
@@ -30,3 +30,4 @@ representation.
 .. autoclass:: urllib3.response.DeflateDecoder
 .. autoclass:: urllib3.response.GzipDecoder
 .. autoclass:: urllib3.response.MultiDecoder
+.. autoclass:: urllib3.response.ZstdDecoder
```

Comment:
I think having ZstdDecoder before MultiDecoder would be nicer.

![image](https://user-images.githubusercontent.com/42327/175360654-1b80275d-b63c-4dda-a53c-7f82ee1ab1fc.png)